app/assets/stylesheets/extend/bootstrap.scss

```diff
@@ -57,3 +57,46 @@
 .result-comment .col-xs-12 {
   float: none;
 }
+
+@media (max-width: 1000px) {
+  .navbar-header {
+    float: none;
+  }
+  .navbar-left,.navbar-right {
+    float: none !important;
+  }
+  .navbar-toggle {
+    display: block;
+    background-color: transparent;
+    background-image: none;
+    border: 1px solid transparent;
+  }
+  .navbar-collapse {
+    border-top: 1px solid transparent;
+    box-shadow: inset 0 1px 0 rgba(255,255,255,0.1);
```

Comment:
Agreed with Hound, try to use one of existing colors, or define a new one in `colors.scss` (and name it using [this nifty tool](http://chir.ag/projects/name-that-color/)).